tests/browser/pages/relying-party.js

```diff
@@ -1,39 +1,66 @@
+const axios = require("axios");
+
 module.exports = class PlaywrightDevPage {
   /**
    * @param {import('@playwright/test').Page} page
    */
   constructor(page, clientId) {
     this.page = page;
-    this.clientId = clientId;
 
-    // Starting URL properties
     const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5050";
     this.baseURL = new URL(websiteHost);
+    this.relyingPartyURL = new URL(process.env.RELYING_PARTY_URL);
+    this.env = process.env.ENV || "dev";
 
-    this.oauthPath = `/oauth2/authorize?request=lorem&client_id=${this.clientId}`;
-    this.startingURL = new URL(this.oauthPath, this.baseURL);
-
-    // Relying Party Return URL
-    this.relyingPartyURL = new URL("http://example.net");
+    if (process.env.MOCK_API === "true") {
+      this.oauthPath = this.getOauthPath("lorem", clientId);
+      this.startingURL = new URL(this.oauthPath, this.baseURL);
+    }
   }
 
   async goto() {
+    if (process.env.MOCK_API === "false") {
```

Comment:
What happens when MOCK_API is not specified at all?

Reply:
I've added a condition to line 17 for if `MOCK_API` is undefined - this lines up with all the other defaults that have already been set so the tests now pass even if no en vars are set